TestON/core/teston.py

```diff
@@ -158,7 +159,7 @@ def componentInit(self,component):
         driver_options ['type'] = driverName
         
         classPath = self.getDriverPath(driverName.lower())
-        driverModule = importlib.import_module(classPath)
+        driverModule = __import__(classPath, globals(), locals(), [driverName.lower()], -1)
```

Comment:
Actually, the like 161 is a recent line I changed and 162 is the original TestON code. I am in the process of fixing some of the things that accidentally got reverted from master in this pull request. 

importlib is a module designed for this purpose and actually uses __import__ internally